src/stream_decoder.c

```diff
@@ -10,11 +10,10 @@
 
 #include "constants.h"
 #include "system_pipeline.h"
-#include "usb.h"
 
 
 #ifndef min
-#define min(a,b)    (((a) < (b)) ? (a) : (b))
+#define min(a,b) (((a) < (b)) ? (a) : (b))
```

Comment:
Since you're the 2nd person to ask I've added comment:

 // Microchip defines this macro in stdlib.h but it's non standard.
// We define it here so that the unit tests work.